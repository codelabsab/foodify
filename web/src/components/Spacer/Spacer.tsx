import React, {Component} from "react";


export default class Spacer extends Component {

  public render() {
    return (<div style={{height: '10em'}}/>);
  }
}
