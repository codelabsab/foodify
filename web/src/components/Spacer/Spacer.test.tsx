import React from 'react';
import { mount } from 'enzyme';
import Spacer from "./Spacer";


it("Should render", async () => {
  let wrapper = mount(<Spacer/>);
  expect(wrapper).toMatchSnapshot();
})
