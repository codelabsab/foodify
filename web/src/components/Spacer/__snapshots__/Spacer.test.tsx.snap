// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 1`] = `
<Spacer>
  <div
    style={
      Object {
        "height": "10em",
      }
    }
  />
</Spacer>
`;
